install_config/install/advanced_install.adoc

```diff
@@ -589,6 +595,84 @@ openshift_master_overwrite_named_certificates=true
 ----
 ====
 
+[[advanced-install-cluster-metrics]]
+== Configuring Cluster Metrics
+
+Cluster metrics are not set to automatically deploy by default. Set the
+following to enable cluster metrics when using the advanced install:
+
+====
+----
+[OSEv3:vars]
+
+openshift_hosted_metrics_deploy=true
+----
+====
+
+=== Metrics Storage
+
+The `*openshift_hosted_metrics_storage_kind*` variable must be set in order to
+use persistent storage. If `*openshift_hosted_metrics_storage_kind*` is not set,
+then cluster metrics data is be stored in an `EmptyDir` volume, which will
+be deleted once the Cassandra pod terminates.
+
+There are three options for enabling cluster storage when using the advanced
+install:
+
+*Option A - NFS Host Group*
+
+When the following variables are set, an NFS volume is be created upon an
+advanced install with path *_nfs_directory/volume_name_* on the host within the
+[nfs] host group. For example, the volume path using these options would be
+*_/exports/metrics_*:
+
+====
+----
+[OSEv3:vars]
+
+openshift_hosted_metrics_storage_kind=nfs
+openshift_hosted_metrics_storage_access_modes=['ReadWriteOnce']
+openshift_hosted_metrics_storage_nfs_directory=/exports
+openshift_hosted_metrics_storage_nfs_options='*(rw,root_squash)'
+openshift_hosted_metrics_storage_volume_name=metrics
+openshift_hosted_metrics_storage_volume_size=10Gi
+----
+====
+
+*Option B - External NFS Host*
+
+To use an external NFS volume, one must already exist with a path of
+*_nfs_directory/_volume_name_* on the storage host.
```

Comment:
should there  be replaceable angle brackets around "nfs_directory" and "volume_name"?